### ISSUE TITLE
update 933161.ra

### ISSUE THREAD

**Issue by fzipi** [coreruleset/coreruleset#3022]
These must be addressed as a group:
- #2684
- #2685
- #3021
- #3022 
 
## Rule 933161

~200 words with short or trivial names, possibly leading to false positives.
```
Examples: 'abs', 'cos'.
```
To mitigate false positives, a regexp matches on function syntax, e.g. 'abs()'.
This rule only runs in paranoia level 3 or higher.

Regexp is generated from function names in regex-assembly/933161.ra

**Comment by dune73** [comment]
Not sure what we have to do here. Doing the list update?

**Comment by fzipi** [comment]
We need to review all PHP functions and redo the classification in the 4 groups. Documenting the process, the decisions on why some words are in PLX, and maybe converge to something that can be done in an automated way.

**Comment by M4tteoP** [comment]
Closing as completed via https://github.com/coreruleset/coreruleset/pull/3273
